server/automute_user_connect.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+func (p *Plugin) updateAutomutingOnUserConnect(userID string) (bool, error) {
+	if !p.isUsersPrimaryPlatformTeams(userID) {
+		return false, nil
+	}
+
+	automuteEnabled, err := p.enableAutomute(userID)
+	if err != nil {
+		p.API.LogError(
```

Comment:
We're about to start raising alerts to SRE for errors. Is this something that we think is human actionable? Or might this fail for other reasons that justifying just a `LogWarn`?

Alternatively, since we already return the error, can we skip the log altogether?

Reply:
Yeah, I can change them to LogWarn since they're not really human actionable as you said.

And returning vs logging is the opposite of my previous comment. I can change that if you think it's a good idea if you think it's better to log these in `api.go` to be a bit more idiomatic